File: app/src/main/java/androidx/preference/ThemePreferenceHelp.java
package androidx.preference;

import android.content.Context;
import android.text.Html;
import android.text.method.LinkMovementMethod;
import android.util.AttributeSet;
import android.widget.TextView;

import com.ubergeek42.WeechatAndroid.R;

public class ThemePreferenceHelp extends HelpPreference {

    public ThemePreferenceHelp(Context context, AttributeSet attrs) {
        super(context, attrs);
    }

    @Override public CharSequence getSummary() {
        StringBuilder sb = new StringBuilder();
        for (String p: ThemeManager.getThemeSearchDirectories(getContext()))
            sb.append("<br>&nbsp;&nbsp;&nbsp;&nbsp;").append(p);

        return Html.fromHtml(getContext().getString(R.string.pref_theme_help, sb));
    }
}

File: app/src/main/java/androidx/preference/ThemePreferenceHelp.kt
package androidx.preference

import android.content.Context
import android.util.AttributeSet
import androidx.core.text.HtmlCompat
import com.ubergeek42.WeechatAndroid.R


class ThemePreferenceHelp(context: Context?, attrs: AttributeSet?) : HelpPreference(context, attrs) {
    override fun getSummary(): CharSequence {
        val indent = "<br>&nbsp;&nbsp;&nbsp;&nbsp;"
        val directories = ThemeManager.getThemeSearchDirectories(context)
        val message = context.getString(R.string.pref__ThemePreferenceHelp__summary,
                indent + directories.joinToString(indent))

        return HtmlCompat.fromHtml(message, HtmlCompat.FROM_HTML_MODE_LEGACY)
    }
}
